### PR TITLE
Test/improve codecoverage

### PR DESCRIPTION
**Pull request by adelkahomolova**
## Description

Write some tests

## Motivation and Context

Improve code coverage

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have added new practice to practice list in README.md.
- [X] I have read the **CONTRIBUTING** document.
- [X] I haven't repeated the code. (DRY)
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,24 +50,24 @@
...
-    "ts-node": "8.6.2",
-    "tslib": "1.11.0",
-    "typescript": "3.8.2",
+    "ts-node": "^8.8.1",
+    "tslib": "^1.11.1",
+    "typescript": "^3.8.3",
     "update-notifier": "4.1.0",
     "xml2js": "0.4.23",
-    "yaml": "1.7.2",
+    "yaml": "^1.8.3",
```
Comment (prokopsimek) [inline package.json]: Remove `^` specifications. We can not allow installing relative version for the end-user.

Comment (prokopsimek) [reply package.json]: btw you can use `npm-check-updates` with command `ncu` to automatically update all versions

File: src/test/factories/responses/gitLab/versionResponseFactory.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import _ from 'lodash';
+import { VersionResponse } from '../../../../services/gitlab/gitlabClient/resources/Version';
+
+export const gitLabVersionResponseFactory = (params?: Partial<VersionResponse>) => {
```
Comment (prokopsimek) [inline src/test/factories/responses/gitLab/versionResponseFactory.ts]:
```suggestion
export const gitLabVersionResponseFactory = (params?: Partial<VersionResponse>): VersionResponse => {
```

File: package.json (hunk 1 above)
Comment (adelkahomolova) [reply package.json]: I know, but it updates also major versions, doesn't it?

Comment (prokopsimek) [reply package.json]: It has an option to update just minor versions.

**Pull request merged**